src/platform/ESP32/bluedroid/BLEManagerImpl.cpp

```diff
@@ -142,7 +142,7 @@ CHIP_ERROR BLEManagerImpl::_Init()
     mRXCharAttrHandle     = 0;
     mTXCharAttrHandle     = 0;
     mTXCharCCCDAttrHandle = 0;
-    mFlags                = kFlag_AdvertisingEnabled;
+    mFlags = CHIP_DEVICE_CONFIG_CHIPOBLE_ENABLE_ADVERTISING_AUTOSTART ? kFlag_AdvertisingEnabled : 0;
```

Comment:
The formatting doesn't seem right. Out of curiosity, do you have clang-format installed? Didn't it align the assignment with the other ones?

Reply:
Done, no I don't, should I just install plugin from VS Code or there is any CHIP-specific clang-format configuration?